roles/kubernetes/master/templates/kubeadm-config.v1alpha1.yaml.j2

```diff
@@ -20,8 +20,11 @@ networking:
   serviceSubnet: {{ kube_service_addresses }}
   podSubnet: {{ kube_pods_subnet }}
 kubernetesVersion: {{ kube_version }}
-{% if cloud_provider is defined and cloud_provider not in ["gce", "oci"] %}
-cloudProvider: {{ cloud_provider }}
+{% if cloud_provider is defined and cloud_provider in ["openstack", "azure", "vsphere", "aws"] %}
+  cloudProvider: {{cloud_provider}}
+  cloudConfig: {{ kube_config_dir }}/cloud_config
```

Comment:
The parameters need to be moved to apiServerExtraArgs.

Reply:
Please provide link to docu for v1alpha1 then. Because it has been like this in v1alpha1 ?

v1alpha2 does indeed have it in the apiServerExtraArgs